### ISSUE TITLE
When user is in the project page, he/she cannot understand what is traceable we should add description or a tip tool short description on that

### ISSUE THREAD

**Issue by DonnaOftadeh** [Giveth/giveth-next#548]

**Comment by DonnaOftadeh** [comment]
![image](https://user-images.githubusercontent.com/53400757/146631075-8fe6441c-6e85-4c71-90be-976f3737e2b0.png)

**Comment by DonnaOftadeh** [comment]
@mo

**Comment by mateodaza** [comment]
I wouldn't know where to put this, maybe we could use help from design

**Comment by laurenluz** [comment]
Can we maybe just change this to say "verified" or "verified and traceable" on the little circle?

when people are looking for verified projects, it is not immediately clear that "traceable" project qualify.

**Comment by MoeNick** [comment]
@markoprljic 
Just mention you here if you can help us with this issue, the problem in general is: Traceables are verified as well, In new design, I see that projects cards can have two lables beside each other. but for this next version,  we don;t have a clue. 
I'm suggest we just add another badge in "Project One Page" and we skip changing the design for this nextjs.

**Comment by laurenluz** [comment]
Maybe for this version, just to make things clearer for users looking to get GIVbacks, we just change Traceable to Verified and then in typescript version have the two labels.

**Comment by MoeNick** [comment]
- [ ] in Projects page: Changing the badge in small circle from Traceable to "Traceable & Verified" 
- [ ]  in "Projects One Page": Add a "Verified" badge to traceable projects. 
@mateodaza pls DM me if it's not clear enough.

**Comment by markoprljic** [comment]
@MoeNick I wouldn't much bother with the design of this, however you guys can do a quick fix is fine IMO. We're focusing on new design soon to be released.

**Comment by MoeNick** [comment]
@markoprljic I agree, just to confirm with you, can we add multiple badges to a project in a new design, it's better we have an example there.

**Comment by markoprljic** [comment]
Yes, that's supported in new design.

**Comment by divine-comedian** [comment]
double badges look to be working! 

![image](https://user-images.githubusercontent.com/67759413/151210966-5598960c-7c1a-4b61-84c3-a422d852fad8.png)
---
![image](https://user-images.githubusercontent.com/67759413/151210649-516d6334-bd3a-4715-bcfe-a175f69d0362.png)

it seems there are two badges - verified and traceable is this intended? I think it looks fine in my opinion. 


the project info page looks to be showing as intended. 


can you confirm this is ok @MoeNick ?
